noir-projects/aztec-nr/address-note/src/address_note.nr

```diff
@@ -23,24 +23,26 @@ struct AddressNote {
 
 impl NoteInterface<ADDRESS_NOTE_LEN, ADDRESS_NOTE_BYTES_LEN> for AddressNote {
 
-    fn compute_nullifier(self, context: &mut PrivateContext) -> Field {
+    fn compute_note_hash_and_nullifier(self, context: &mut PrivateContext) -> [Field; 2] {
```

Comment:
I'd return a tuple with `(note_hash_for_nullify, nullifier)` instead of an array - it is less cryptic.

Reply:
Good idea. Addressed in later commits